dbt_core_integration.py

```
@@ -1,3 +1,9 @@
+try:
+    from dbt.version import __version__ as dbt_version
+except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)